cmd/kots/cli/admin-console-upgrade.go

```diff
@@ -151,3 +160,43 @@ func AdminConsoleUpgradeCmd() *cobra.Command {
 	cmd.Flags().Bool("skip-rbac-check", false, "set to true to bypass rbac check")
 	return cmd
 }
+
+func isDowngrade(ctx context.Context, clientset kubernetes.Interface, namespace string) (string, bool) {
+	if strings.Contains(buildversion.Version(), "nightly") {
+		return "", false
+	}
+
+	containers := []corev1.Container{}
+
+	s, err := clientset.AppsV1().StatefulSets(namespace).Get(ctx, "kotsadm", metav1.GetOptions{})
+	if err == nil {
+		containers = s.Spec.Template.Spec.Containers
+	} else {
+		d, err := clientset.AppsV1().Deployments(namespace).Get(ctx, "kotsadm", metav1.GetOptions{})
+		if err == nil {
+			containers = d.Spec.Template.Spec.Containers
+		}
+	}
+
+	if len(containers) == 0 {
+		return "", false
+	}
+
+	parts := strings.Split(containers[0].Image, ":")
+	if len(parts) != 2 {
```

Comment:
There can be 3 parts if host name is followed by port number:  `parts[len(parts)-1]`